src/NUnitFramework/nunitlite.runner/TextUI.cs

```diff
@@ -342,20 +342,25 @@ public void DisplaySummaryReport(ResultSummary summary)
                 SkipLine();
 
             WriteSectionHeader("Test Run Summary");
-            WriteLabelLine("   Overall result: ", overallResult, overallStyle);
+            WriteLabelLine("  Overall result: ", overallResult, overallStyle);
 
-            WriteSummaryCount("   Tests run: ", summary.RunCount);
+            WriteSummaryCount("  Tests run: ", summary.RunCount);
```

Comment:
Let's make this Test Count: or Test Cases: and use the total of test cases as the value.